HW_04/task2-3.py

```diff
@@ -0,0 +1,18 @@
+input_text = input("Введіть тестові дані:")
+if str.isdigit(input_text):
+    if (int(input_text) % 2) == 0:
```

Comment:
nit: brackets could be omitted
```suggestion
    if int(input_text) % 2 == 0:
```